var/spack/repos/builtin/packages/py-pyqt5/package.py

```diff
@@ -12,11 +12,12 @@ class PyPyqt5(SIPPackage):
     Windows, OS X, Linux, iOS and Android. PyQt5 supports Qt v5."""
 
     homepage = "https://www.riverbankcomputing.com/software/pyqt/intro"
-    url      = "https://www.riverbankcomputing.com/static/Downloads/PyQt5/5.13.0/PyQt5_gpl-5.13.0.tar.gz"
-    list_url = "https://www.riverbankcomputing.com/software/pyqt/download5"
+    url = "https://files.pythonhosted.org/packages/source/P/PyQt5/PyQt5-5.15.2.tar.gz"
+    list_url = "https://pypi.org/simple/PyQt5/"
 
     sip_module = 'PyQt5.sip'
 
+    version('5.15.6', sha256='80343bcab95ffba619f2ed2467fd828ffeb0a251ad7225be5fc06dcc333af452')
     version('5.13.1', sha256='54b7f456341b89eeb3930e786837762ea67f235e886512496c4152ebe106d4af')
     version('5.13.0', sha256='0cdbffe5135926527b61cc3692dd301cd0328dd87eeaf1313e610787c46faff9')
     version('5.12.3', sha256='0db0fa37debab147450f9e052286f7a530404e2aaddc438e97a7dcdf56292110')
```

Comment:
These older versions aren't available on PyPI. I think we'll need a `url_for_version` here

Reply:
The problem here is that the sources of the older versions are nor available on the original source path anymore (meaning <https://riverbankcomputing.com/software/pyqt/download> refers now to pypi) and on pypi there are only wheels for them. That's what I meant with my comment above that they can only be found via the spack mirror.

Reply:
In that case you should probably deprecate those older versions so we can remove them after the next Spack release